src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/MeterFactoryExtensions.cs

```diff
@@ -3,9 +3,8 @@
 
 using System;
 using System.Collections.Generic;
-using System.Diagnostics.Metrics;
 
-namespace Microsoft.Extensions.Diagnostics.Metrics
+namespace System.Diagnostics.Metrics
```

Comment:
Do we have tests for MeterFactoryExtensions.Create somewhere?

Reply:
Yes it is in https://github.com/dotnet/runtime/blob/f9f7f8079377039a8d42269434a41ca55d14d591/src/libraries/Microsoft.Extensions.Diagnostics/tests/MetricsTests.cs#L43